Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -64,6 +64,12 @@
 #import <UIKit/UIColor.h>
 #endif
 
+// FIXME: We should rationalize these with the values in ViewGestureController.
+// For now, we'll leave them differing as they do in PDFPlugin.
+static const CGFloat minimumZoomScale = 0.2;
+static const CGFloat maximumZoomScale = 6.0;
+static const CGFloat zoomIncrement = 1.18920;
```

Comment:
Weird

Reply:
There's a note in PDFKit about how this is weird as well!